purchase_shop/views/purchase_order_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form_inherit" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="priority" eval="21" />
+        <field name="arch" type="xml">
+            <field name="partner_ref" position="after">
+                <field name="shop_id" />
+            </field>
+            <xpath expr="//field[@name='incoterm_id']" position="after">
+                <field name="purchased_by_id" />
+            </xpath>
+            <xpath expr="//sheet/group" position="inside">
+                <group>
+                    <field name="employee_id" />
+                </group>
+            </xpath>
+        </field>
+    </record>
+    <record id="purchase_order_tree" model="ir.ui.view">
```

Comment:
I cannot see this view in the tree view of the purchase.order. Can you please confirm, the same applies to the search view.

Reply:
I see that the `purchase_order_tree` exist in V15, but not shown in the purchase moudule. Looks like we need to adjust the `purchase_order_view_tree`.